cmd/acra-translator/acra-translator.go

```diff
@@ -162,6 +172,56 @@ func main() {
 		os.Exit(0)
 	})
 
+	go sigHandlerSIGHUP.RegisterWithContext(mainContext)
+	sigHandlerSIGHUP.AddCallback(func() {
+		log.Infof("Received incoming SIGHUP signal")
+		log.Debugf("Stop accepting new connections, waiting until current connections close")
+		readerServer.StopListeners()
+
+		// Set env flag for forked process
+		if err := os.Setenv(GracefulEnv, "true"); err != nil {
+			log.WithError(err).Errorln("Unexpected error on os.Setenv, graceful restart won't work. Please check env variables, especially gracefulEnv")
+		}
+
+		var fdHTTP, fdGRPC uintptr
+		if listener := readerServer.GetHTTPListener(); listener != nil {
+			fdHTTP, err = network.ListenerFileDescriptor(readerServer.GetHTTPListener())
+			if err != nil {
+				log.WithError(err).WithField(logging.FieldKeyEventCode, logging.EventCodeErrorCantGetFileDescriptor).
+					Fatalln("System error: failed to get acra-socket file descriptor for HTTP:", err)
+
+			}
+		}
+		if listener := readerServer.GetGRPCListener(); listener != nil {
+			fdGRPC, err = network.ListenerFileDescriptor(readerServer.GetGRPCListener())
+			if err != nil {
+				log.WithError(err).WithField(logging.FieldKeyEventCode, logging.EventCodeErrorCantGetFileDescriptor).
+					Fatalln("System error: failed to get acra-socket file descriptor for GRPC:", err)
```

Comment:
What is "acra-socket"?
I mean from user perspective. What users should think about when they read "failed to get acra-socket file descriptor ..". Maybe use just "failed to get Acra file descriptor .."?

Reply:
legacy from `AcraServer`. Yes `failed to get Acra gRPC file descriptor` is really better